go.sh

```diff
@@ -19,10 +20,10 @@ for CMD in flockerctl flocker-ca flocker-deploy flocker-config flocker-install f
     cat <<EOF |sudo tee /usr/local/bin/${PREFIX}${CMD} >/dev/null
 #!/bin/sh
 DEPRECATED="${DEPRECATED}"
-read -d '' DEPRECATION_WARNING <<END_WARNING
+DEPRECATION_WARNING=\$(cat <<END_WARNING
```

Comment:
No, this is the content of the wrapper script. So most of the variables need to be escaped.